packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -50,9 +50,17 @@ export class CoreAppsService {
   preboot(corePreboot: InternalCorePreboot, uiPlugins: UiPlugins) {
     this.logger.debug('Prebooting core app.');
 
+    this.logger.info(
+      `!!!!!!!!!!!!Will register preboot routes: ${uiPlugins.public.size > 0}!!!!!!!!!!!!!`
+    );
+
+    this.logger.info(
+      `!!!!!!!!!!Found ${Array.from(uiPlugins.public.keys()).join(', ')} uiPlugins!!!!!!!!!!`
+    );
+
     // We register app-serving routes only if there are `preboot` plugins that may need them.
     if (uiPlugins.public.size > 0) {
-      this.registerPrebootDefaultRoutes(corePreboot, uiPlugins);
+      // this.registerPrebootDefaultRoutes(corePreboot, uiPlugins);
```

Comment:
Should this be uncommented to test that the config `interactiveSetup.enabled: false` doesn't call this?

Reply:
yeah with your changes to the stack packs we should be able to remove this and the change to `kibana.yml`

Reply:
I just pushed a commit reverting those changes